### ISSUE TITLE
[Feature Request] Add "Wiki" commands in Azure DevOps CLI extension

### ISSUE THREAD

**Issue by snankani** [Azure/azure-devops-cli-extension#435]
As a user of Azure DevOps CLI, I should be able to create, delete, publish code wiki and other wiki related actions from CLI.

[Updated - atbagga]

|  Command | Description  	|   	
|---	|---	|	
| az devops wiki create   	|   Create/Publish a new wiki |   
| az devops wiki delete 	|   Delete a wiki |     	
| az devops wiki list |  List all wikis in an organization or project	|   
| az devops wiki show            |  Show details about a wiki or Open in browser |   
| az devops wiki page create |  Create a new wiki page |
| az devops wiki page delete |  Delete a wiki page |
| az devops wiki page update |  Update content of a wiki page or Open in browser |
| az devops wiki page show |  Show details of a wiki page |

```
Group
    az devops wiki : Manage wikis.

Subgroups:
    page   : Manage wiki pages.

Commands:
    create : Create a wiki.
    delete : Delete a wiki.
    list   : List all the wikis in a project or organization.
    show   : Show details of a wiki.
```

```

Group
    az devops wiki page : Manage wiki pages.

Commands:
    create : Add a new page.
    delete : Delete a page.
    show   : Get the content of a page or open a page.
    update : Edit a page.
```

**Comment by geverghe** [comment]
Including relevant excerpts from discussion to provide more context:

## Use Cases
Couple of things we identified in a short brainstorming session were-
1.	Publishing a new wiki version (Low usage use-case)
2.	Publish Code wikis from different folders
3.	Automatically create a sprint summary page (This can be a script which collects build/PR summaries and creates a wiki page at the end of iteration with this info.)
4.	Support template pages through CLI. For example: az devops wiki page add –template SprintRetro -path ./azdev.in/protocol/

## Open Questions
1. Interesting, would wiki commands go under devops or repos since wiki is a git repo.

**Comment by gauravsaralMs** [comment]
>> 1. Interesting, would wiki commands go under devops or repos since wiki is a git repo.

Implementation wise Wiki is using Git for outside customer it does not matter. IMO wiki should go under DevOps because with verticals it is possible that someone does not have repos but still have wiki

**Comment by vbossica** [comment]
To publish code to a wiki (or any Git repository), the extension needs to support Git `pushes` commands. They are already part of an old PR but I'll be moving the commands to the new code base shortly.

**Comment by atbagga** [comment]
Wiki commands as stated in description will be available in the next release 0.5.0
